src/Symfony/Bundle/FrameworkBundle/Tests/Logger/LoggerTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Logger;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+use Symfony\Bundle\FrameworkBundle\Logger\Logger;
+
+/**
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ */
+class LoggerTest extends TestCase
+{
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var string
+     */
+    private $tmpFile;
+
+    public function setUp()
+    {
+        $this->tmpFile = sys_get_temp_dir().'/log';
+        $this->logger = new Logger(LogLevel::DEBUG, $this->tmpFile);
+        file_put_contents($this->tmpFile, '');
```

Comment:
We should at least try to clean up. In other tests saving files on the filesystem we use `tearDown` to remove them.